sdk/keyvault/keyvault-certificates/test/CRUD.test.ts

```diff
@@ -279,13 +279,24 @@ describe("Certificates client - create, read, update and delete", () => {
       basicCertificatePolicy,
       testPollerProperties
     );
+
+    // Using the poller
```

Comment:
What do you think about converting `it("can get a deleted certificate"` to `describe("can get a deleted certificate"` and having two `it` functions, one that uses the poller, and one that uses `getDeletedCertificate`?

To me that brings attention to the intent of what you're testing (which I assume is that both methods work), and you can reduce the chance that the state of one function affects the other. I also generally like having individual tests test _one_ thing when possible :smile:.

Reply:
this feedback is definitely outside of the scope of this PR, but this feedback aligns with my goal of making better tests in general. I scheduled a meeting with Jeff on Monday to brainstorm with him about how to improve our tests. Would you like to be invited?

Reply:
oh my bad, I thought I was in another PR. I will address your feedback in this PR!

Reply:
Though the invitation for Monday still stands!!